tests/acceptance/dns_test.go

```diff
@@ -203,36 +174,22 @@ var _ = Describe("DNS", func() {
 			return records[0].String(), nil
 		}).Should(Equal("1.2.3.5"))
 
-		By("creating an A record for the ingress")
-		Eventually(func() error {
-			var err error
-			records, err = resolver.LookupIP(ctx, "ip", ingressDomain)
-			return err
-		}).Should(Succeed())
-
-		Expect(records).To(HaveLen(1))
-		Expect(records[0].String()).To(Equal("10.0.0.2"))
```

Comment:
The A record for the ingress is created by external-dns, we don't need to test it here